packages/design-system/src/components/FieldError/FieldError.tsx

```diff
@@ -4,25 +4,25 @@ import { AlertCircleIcon } from '../Icons/AlertCircleIcon';
 import { t } from '../i18n';
 
 /**
- * <InlineError> is an internal component used by <FormLabel>
- * <InlineError> is also exported for advanced design system use cases, where the internal component can be leveraged to build custom form components
+ * <FieldError> is an internal component used by <FormLabel>
+ * <FieldError> is also exported for advanced design system use cases, where the internal component can be leveraged to build custom form components
  */
 
-interface InlineErrorProps {
+export interface FieldErrorProps {
   children?: React.ReactNode;
   className?: string;
   id?: string;
   inversed?: boolean;
 }
 
-export function InlineError({
+export function FieldError({
   children,
   className,
   id,
   inversed,
-}: InlineErrorProps): React.ReactElement {
+}: FieldErrorProps): React.ReactElement {
   const classes = classNames(
-    'ds-c-inline-error',
+    'ds-c-inline-error', // For backwards compatibility
```

Comment:
is it worth adding the same deprecation comment here as well?

Reply:
Just that we don't use that class to style anything anymore?

Reply:
yeah. we comment in the inline error file that this class will be deprecated, but don't have any sort of TODO here to actually deprecate it.